lib/fluent/plugin/filter_pan_anonymizer.rb

```diff
@@ -20,8 +20,13 @@ def initialize
     def configure(conf)
       super
 
+      formats = conf.each_element.map do |i|
```

Comment:
Isn't enough to check `@pan_configs`? I think `formats` directives are only in `<pan>` section.

Reply:
Ah, fluentd can't parse a regexp array correctly.